docker-compose.yml

```diff
@@ -20,15 +22,35 @@ services:
     ports:
       - ${SERVER_PORT}:${SERVER_PORT}
     env_file: .env
+    networks:
+      - krs_app
     volumes:
       - ./server:/server
       - krs-server-node-modules-vol:/server/node_modules
     depends_on:
       postgres-db:
         condition: service_healthy
 
+  client:
+    container_name: ${CLIENT_HOST}
+    build: ./client
+    ports:
+      - ${CLIENT_PORT}:${CLIENT_PORT}
+    env_file: .env
+    networks:
+      - krs_app
+    volumes:
+      - ./client:/client
+      - krs-client-node-modules-vol:/client/node_modules
+
+networks:
```

Comment:
wydaje mi się że domyślnie jest 1 sieć type 'bridge' do której wszystkie kontenery są podpięte. Coś nie działało, że to dodałeś?